charts/incubator/fancyindex/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+kubeVersion: ">=1.16.0-0"
+name: fancyindex
+version: 1.0.1
+appVersion: "latest"
+description: fancyindex App for TrueNAS SCALE
+type: application
+deprecated: false
+home: https://github.com/truecharts/apps/tree/master/charts/incubator/fancyindex
+icon: https://raw.githubusercontent.com/truecharts/apps/master/incubator/fancyindex/icon.png?raw=true
+keywords:
+  - fancyindex
+sources:
+  - https://github.com/truecharts/apps/tree/master/charts/incubator/fancyindex
```

Comment:
Add also as source the actual repository and/or dockerhub that this app is coming from.